CMakeLists.txt

```diff
@@ -275,6 +275,10 @@ if(NOT MAKE_DOCS_ONLY)
     endif()
 
     if(MAKE_DOCS)
```

Comment:
Because you used to be able to make docs alongside the binaries.  I put up a jira to fix that functionality